### PR TITLE
feat(hss/group): add new host group resource

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
add new resource for HSS service.
20 minutes maybe too short...

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add new resource support.
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/hss' TESTARGS='-run=TestAccHostGroup_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/hss -v -run=TestAccHostGroup_basic -timeout 360m -parallel 4
=== RUN   TestAccHostGroup_basic
=== PAUSE TestAccHostGroup_basic
=== CONT  TestAccHostGroup_basic
--- PASS: TestAccHostGroup_basic (788.80s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/hss       789.171s
```

### CODE REVIEW

File: docs/resources/hss_host_group.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region where the host group is located.
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the name of the host group.  
+  The valid length is limited from `1` to `64`, only chinese and english letters, digits, hyphens (-), underscores (_)
```
Comment (Jason-Zhang9309) [inline docs/resources/hss_host_group.md]: s/chinese/Chinese，s/english/English

File: huaweicloud/services/hss/resource_huaweicloud_hss_host_group.go [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+			"host_ids": {
+				Type:        schema.TypeSet,
+				Optional:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Description: "The list of host IDs.",
+			},
+			"enterprise_project_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The ID of the enterprise project to which the host group belongs.",
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/hss/resource_huaweicloud_hss_host_group.go]: missing ForceNew

File: docs/resources/hss_host_group.md (hunk 1 above)
Comment (Lance52259) [reply docs/resources/hss_host_group.md]: done.

File: huaweicloud/services/hss/resource_huaweicloud_hss_host_group.go (hunk 2 above)
Comment (Lance52259) [reply huaweicloud/services/hss/resource_huaweicloud_hss_host_group.go]: fixed.

File: huaweicloud/services/hss/resource_huaweicloud_hss_host_group.go [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+	}
+	return nil, fmt.Errorf("invalid host group list, want 'hssv5model.HostGroupItem', but '%T'.", result[0])
+}
+
+func resourceHostGroupRead(_ context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	var (
+		cfg     = meta.(*config.Config)
+		region  = cfg.GetRegion(d)
+		groupId = d.Id()
+		// epsId   = cfg.DataGetEnterpriseProjectID(d)
```
Comment (ShiChangkuo) [inline huaweicloud/services/hss/resource_huaweicloud_hss_host_group.go]: please delete this line

File: huaweicloud/config/hc_config.go [hunk 4]
```diff
@@ -277,6 +278,15 @@ func (c *Config) HcDmsV2Client(region string) (*dmsv2.KafkaClient, error) {
 	return dmsv2.NewKafkaClient(hcClient), nil
 }
 
+// HcDmsV2Client is the DMS service client using huaweicloud-sdk-go-v3 package
```
Comment (ShiChangkuo) [inline huaweicloud/config/hc_config.go]: the comment is not correct

File: huaweicloud/services/hss/resource_huaweicloud_hss_host_group.go (hunk 3 above)
Comment (Lance52259) [reply huaweicloud/services/hss/resource_huaweicloud_hss_host_group.go]: done.

File: huaweicloud/config/hc_config.go (hunk 4 above)
Comment (Lance52259) [reply huaweicloud/config/hc_config.go]: fixed.

**Pull request merged**